bonsai/Modules/Categories/CategoriesContainerView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CategoriesContainerView.swift
+//  bonsai
+//
+//  Created by Максим Алексеев  on 10.12.2021.
+//
+
+import SwiftUI
+
+struct CategoriesContainerView: View {
+    @State var selectedIndex = 0
+    
+    init() {
+        UINavigationBar.appearance().largeTitleTextAttributes = [.foregroundColor: UIColor.white]
+        UINavigationBar.appearance().titleTextAttributes = [.foregroundColor: UIColor.white]
+        
+        UINavigationBar.appearance().tintColor = UIColor(BonsaiColor.orange)
```

Comment:
why orange? :)

Reply:
because of back button color in nav bar

Reply:
Fixed in https://github.com/appbonsai/bonsai./pull/20